spec/rmagick/draw_spec.rb

```diff
@@ -0,0 +1,157 @@
+class Magick::Draw
+  def self._dummy_img_
+    @@_dummy_img_
+  end
+end
+
+RSpec.describe Magick::Draw do
+  let(:draw) { Magick::Draw.new }
+
+  describe '#_test_dummy_img_' do
```

Comment:
By using a `#` at the start of the method name, you're implying that it's an instance method, when it's a class method on `Magick::Draw`. Please, change the `#` to a `.`.